modules/docs/mdx/7.0-MIGRATION-GUIDE.mdx

```diff
@@ -154,6 +155,11 @@ If your code contains any hacks to make a `Modal` overflow, these hacks should n
 `max-height` of the `Modal.Body` element using calculations. These should be removed. The
 `Popup.Card` now has a max height and the `Popup.Body` height is automatically calculated.
 
+## Popper Props Update
+
+We removed the `containerElement` prop from Popper component because it's no longer needed with
+Fullscreen API.
```

Comment:
do we have a codemod for this?

Reply:
I've checked usage of Popper, so it will be good to add codemod for making migration easier. Thanks for catching it, I'll add it